src/marqo/tensor_search/models/external_apis/s3.py

```diff
@@ -0,0 +1,17 @@
+from pydantic.dataclasses import dataclass
+from typing import Optional
+from marqo.tensor_search.models.external_apis.abstract_classes import (
+    ObjectLocation, ExternalAuth
+)
+
+
+class S3Auth(ExternalAuth):
+    aws_secret_access_key: str
+    aws_access_key_id: str
+    aws_session_token: Optional[str] = None
+
```

Comment:
why the HFAuth inherits ObjectionLocation but S3Auth inherits ExternalAuth? typo i guess?

Reply:
fixed! Thanks!